### ISSUE TITLE
bug: [On MacOS] the tests do not run when the file is updated, they run on the tests being updated. 

### ISSUE THREAD

**Issue by O4FDev** [davidjrice/autopytest#2]
OS: MacOS 14.0 (23A344)
Project link: https://github.com/O4FDev/sample_py_app-testing
Python version: Python 3.11.5

pip freeze:

PyYAML @ file:///Users/runner/miniforge3/conda-bld/pyyaml_1695373551341/work
watchdog @ file:///Users/runner/miniforge3/conda-bld/watchdog_1695395480324/work

This is on a version of autopytest installed from the Make command wo/changes to the source code.

**Comment by O4FDev** [comment]
pr for this: https://github.com/davidjrice/autopytest/pull/3

**Comment by davidjrice** [comment]
Hey this was actually a few things:

* the tests should run on test file update, that's a feature!
* I have added configuration support for not including the source dir in the test lookup which works with your example project
* Also, yep. There was an issue with support on OSX as fsevent uses absolute paths.

All of these should be addressed in [0.0.5](https://github.com/davidjrice/autopytest/releases/tag/0.0.5). Tested out with your example project and will be working for you. Let me know if there are any issues!
